pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -16,7 +16,7 @@ DO $$
 DECLARE
    new_path text;
 BEGIN
-   new_path := current_setting('search_path') || format(',%L,%L', 'SCHEMA_PROM', 'SCHEMA_METRIC');
+   new_path := current_setting('search_path') || format(',%L,%L,%L,%L', 'SCHEMA_EXT', 'SCHEMA_PROM', 'SCHEMA_METRIC', 'SCHEMA_CATALOG');
```

Comment:
can we add a comment about why the order is the way it is and how this interacts with extensions.